README.md

```diff
@@ -1,73 +1,122 @@
+# 🏗 Scaffold-ETH 2 PWA 📱
 
-⚙️ Built using NextJS, RainbowKit, Hardhat, Wagmi, and Typescript.
+This **forkable** project provides the infraestructure to build a Progressive Web App (PWA) using Scaffold-ETH 2 base features, plus PWA oriented extra features, like Push Notifications and the capability to Install the PWA on your device.
 
-- ✅ **Contract Hot Reload**: Your frontend auto-adapts to your smart contract as you edit it.
-- 🔥 **Burner Wallet & Local Faucet**: Quickly test your application with a burner wallet and local faucet.
-- 🔐 **Integration with Wallet Providers**: Connect to different wallet providers and interact with the Ethereum network.
+To learn more about base Scaffold-ETH 2 features and development guide, check our [Docs](https://docs.scaffoldeth.io/) and [Website](https://scaffoldeth.io/).
 
-![Debug Contracts tab](https://github.com/scaffold-eth/scaffold-eth-2/assets/55535804/1171422a-0ce4-4203-bcd4-d2d1941d198b)
+## 🏃 Quick Start
 
-## Requirements
+To get started with Scaffold-ETH 2 PWA, follow the steps below:
 
-Before you begin, you need to install the following tools:
+### 1 . Clone this repo & install dependencies
 
-- [Node (v18 LTS)](https://nodejs.org/en/download/)
-- Yarn ([v1](https://classic.yarnpkg.com/en/docs/install/) or [v2+](https://yarnpkg.com/getting-started/install))
-- [Git](https://git-scm.com/downloads)
+```bash
+gh repo clone BuidlGuidl/PWA-burner-wallet
+cd PWA-burner-wallet
+yarn install
+```
 
-## Quickstart
+### 2 . Setting up Firebase
 
-To get started with Scaffold-ETH 2, follow the steps below:
+> Note : You can also use other database as well, we are using Firebase for this example checkout `packages/nextjs/database/firebase`.
 
-1. Clone this repo & install dependencies
+Copy `packages/nextjs/.env.example` into `packages/nextjs/.env` file and fill in your Firebase credentials which starts with `FB`.
 
-```
-git clone https://github.com/scaffold-eth/scaffold-eth-2.git
-cd scaffold-eth-2
-yarn install
+Checkout [this article](https://softauthor.com/add-firebase-to-javascript-web-app/#add-firebase-sdk-to-javascript-web-app) on how to get config variables from Firebase.
+
+> Hint: If it's your first time using Firebase, please read the article from the beginning, as you'll need to create a Firebase project and register a web app first.
+
+We use the Firebase config variables in our `packages/nextjs/database/firebase/config` file:
+
+```js
+// Your web app's Firebase configuration
+const firebaseConfig = {
+  apiKey: process.env.FB_API_KEY,
+  authDomain: process.env.FB_AUTH_DOMAIN,
+  projectId: process.env.FB_PROJECT_ID,
+  storageBucket: process.env.FB_STRG_BUCKET,
+  messagingSenderId: process.env.FB_MESSAGING_SENDER_ID,
+  appId: process.env.FB_APP_ID,
+};
 ```
 
```

Comment:
Suggestion for this part (I think we can simplify and not sure about linking a weird article :D)

----

Create your [Firebase project](https://console.firebase.google.com/) and register a web app. It'll give you a config object that looks like this:

```js
// Your web app's Firebase configuration
const firebaseConfig = {
  apiKey: "...",
  authDomain: "...",
  projectId: "...",
  storageBucket: "...",
  messagingSenderId: "...",
  appId: "...",
};
```

Copy `packages/nextjs/.env.example` into `packages/nextjs/.env` file and fill in your Firebase credentials which starts with `FIREBASE_`.

```

Reply:
Yep, that section is probably the most confusing part of the setup and has to be simplified and explained better 🙌

I've just pushed the simplified version in this commit: https://github.com/BuidlGuidl/PWA-burner-wallet/pull/1/commits/4af299ac1d8abcd727f7524e546a89b49cb5d9f1